### PR TITLE
Added a prop for light mode in the tooltip

### LINKED ISSUE #9: Add a prop for light mode in the tooltip
Currently, the tooltip has a light text on a dark background by default. Add a prop makes the tooltip to have a light background with dark text colour

### PR DESCRIPTION
**Pull request by ankiiitraj**
Fixes #9 
Gif demonstration for changes!
ezgif com-video-to-gif(1)

### CODE REVIEW

**Comment by ankiiitraj** [comment]
@prateek3255 this is ready for review!

File: src/lib/index.tsx [hunk 1]
```diff
@@ -62,6 +68,7 @@ const CopyMailTo = ({
   anchorStyles = {},
 }: {
   email: string;
+  theme?: string;
```
Comment (prateek3255) [inline src/lib/index.tsx:71]: Use [TypeScript String Literals](https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types) here for defining the types of theme.

Comment (ankiiitraj) [reply src/lib/index.tsx]: Resolved! Thanks for the suggestion @prateek3255

**Comment by ankiiitraj** [comment]
@prateek3255 Ready for the review! Mentioning for visibility! 😃

File: src/lib/index.tsx [hunk 2]
```diff
@@ -25,6 +25,11 @@ const containerBaseStyles: React.CSSProperties = {
   position: "relative",
 };
 
+const light: React.CSSProperties = {
+  backgroundColor: "white",
+  color: "black",
+}
+
 const tooltipBaseStyles: React.CSSProperties = {
```
Comment (prateek3255) [inline src/lib/index.tsx:33]: Make this a function that accepts theme instead of creating another variable.

**Review by prateek3255 — CHANGES_REQUESTED** [review]

File: src/lib/index.tsx (hunk 2 above)
Comment (ankiiitraj) [reply src/lib/index.tsx:33]: You mean a function that takes theme as input and returns the properties of the theme.
in this case, if theme is light function should return {backgrondColor: "white", color: "black"}

Comment (prateek3255) [reply src/lib/index.tsx:33]: Yes check #12, also create a type for theme similar to #12

Comment (ankiiitraj) [reply src/lib/index.tsx]: Done! :)

**Comment by prateek3255** [comment]
Thanks for working on this @ankiiitraj

**Comment by prateek3255** [comment]
@all-contributors please add @ankiiitraj for code

**Pull request merged**